src/pymorize/units.py

```diff
@@ -0,0 +1,99 @@
+"""
+This module deals with the auto-unit conversion in the cmorization process.
+In case the units in model files differ from CMIP Tables, this module attempts to
+convert them automatically.
+
+In case of missing units in either model files or CMIP Tables, this module can
+not convert from a dimentionless base to something with dimension. Dealing with
+such thing have to done with `action` section in the Rules module on a per
+variable basis.
+"""
+
+import re
+from typing import Pattern
+
+import pint
+from chemicals import periodic_table
+
+
+ureg = pint.UnitRegistry()
+# ureg.define('degC = degree_Celsius')
+# https://ncics.org/portfolio/other-resources/udunits2/
+#ureg.define("degrees_east = deg")
+#ureg.define("degree_east = deg")
+ureg.define("degrees_north = deg")
+ureg.define("degree_north = deg")
+#ureg.define("degrees_west = -1 * deg")
+#ureg.define("degrees_south = -1 * deg")
```

Comment:
You could leave this in if you they are useful, my earlier comment was regarding the chemical weights.